index.html

```diff
@@ -186,6 +186,48 @@ <h1>Introduction</h1>
 market competition, reduced vendor lock-in and switching costs.
     </p>
 
+    <p>
+This specification contains the following sections that software architects
+and implementers might find useful:
+    </p>
+
+    <ul class="bullet">
+      <li>
+<a href="#design-goals"></a> specifies the high level design goals
+that drove the formulation of this specification.
+      </li>
+      <li>
+<a href="#architecture-overview"></a> highlights the different roles
+and components that are contemplated by the architecture.
+      </li>
+      <li>
+<a href="#terminology"></a> defines specific terms that are used throughout
+the document.
+      </li>
+      <li>
+<a href="#authorization"></a> elaborates upon the various forms of authorization
+that can be used with the API.
+      </li>
+      <li>
+<a href="#issuing"></a> describes both the internal and external APIs for
+issuing <a>verifiable credentials</a> as well as updating their status.
+      </li>
+      <li>
+<a href="#verifying"></a> specifies the internal APIs for verifying both
+<a>verifiable credentials</a> and verifiable presentations.
+      </li>
+      <li>
+<a href="#presenting"></a> defines both the internal and external APIs for
```

Comment:
I think `internal` and `external` should be avoided here, similar to how we've moved away from this terminology while working on the spec. If not removed, something should be said here about what these words mean in this context; if removed (as I would prefer), they should be replaced with something like `within trust domain` and `across trust domains`.

Reply:
"within" and "across" are better. I was struggling to find better words.

Reply:
@TallTed, I applied your suggestions in 93b26f2ee8b3dc3e7276eb169baac8c706ce3ee4.